src/components/greetings.jsx

```diff
@@ -0,0 +1,24 @@
+import { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { connect, useSelector } from 'react-redux';
+import fetchGreeting from '../redux/actions';
+
+function Greeting({ fetchGreeting }) {
+  const greeting = useSelector((state) => state.greeting);
+
+  useEffect(() => {
+    fetchGreeting();
+  }, [fetchGreeting]);
+
+  return <div>{greeting}</div>;
+}
+
+Greeting.propTypes = {
+  fetchGreeting: PropTypes.func.isRequired,
+};
+
+const mapDispatchToProps = {
+  fetchGreeting,
+};
+
```

Comment:
- [ ] Good job on using the `useSelector` for getting your greeting.  As told by previous reviewer still the gretting is undefined and the page is black. please fix this issue and ask for review.
### Error
![Screenshot 2023-11-12 072304](https://github.com/MasumaJaffery/Hello-React-Frontend/assets/81727988/777c01e3-3c17-407e-8d1f-f313f4d8573f)